Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift

```diff
@@ -97,6 +109,14 @@ class GenerateKeyFlow: AddKeyFlow {
         push(flow: replaceOwnerFlow)
     }
 
+    func didInviteToAddKey(openKeyDetails: Bool = true) {
```

Comment:
I think this method is not needed, you could use the `didAddKeyAsOwner`. What is the reason that this duplicate method exists?